docs/Running-behind-nginx.md

```diff
@@ -114,25 +152,25 @@ Of course you can add as many backend servers as you like.
 
 Using the above, you access Netdata on the backend servers, like this:
 
-- `http://nginx.server/netdata/server1/` to reach `backend-server1`
-- `http://nginx.server/netdata/server2/` to reach `backend-server2`
+- `http://netdata.example.com/netdata/server1/` to reach `backend-server1`
+- `http://netdata.example.com/netdata/server2/` to reach `backend-server2`
 
 ### Using TLS communication
```

Comment:
Chang to: Encrypt the communication between nginx and netdata.
Thiago had comments on how this should be written, I'll let him review the new statements.

Reply:
Hi @cakrit the encrypt communication was changed according my initial suggestion

Reply:
'Encrypt the communication between Nginx and Netdata' looks better than just 'Using TLS communication'

@thiagoftsm you suggested its content.

@cakrit I have updated the header title.

Reply:
Your sentence let the things more explicit, but there are different ways of encrypt a channel, TLS is one of them, considering that we are already using the TLS in the description, I do not see any problem in change the title.